packages/@aws-cdk/aws-codepipeline/test/pipeline.test.ts

```diff
@@ -566,3 +859,57 @@ class ReusePipelineStack extends cdk.Stack {
     });
   }
 }
+
+const createPipeline = (scope: Construct, pipelineName?: string, pipelineId: string = 'Pipeline') => {
```

Comment:
I feel like this name is not great for this function. What pipeline does it create? Clearly it has some structure (two stages with source and build actions), and some specific settings passed. Would be awesome if the name reflected some of that.

Reply:
Will rename.